.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        target:
+          - thumbv7em-none-eabi
+          - thumbv6m-none-eabi # like the Cortex-M0+
+          - x86_64-unknown-linux-gnu
+          - x86_64-pc-windows-msvc
+
+    steps:
+      - name: Install build dependencies
+        shell: bash
+        run: |
+          env && pwd && sudo apt-get update -y -qq && sudo apt-get install -y -qq llvm libc6-dev-i386 libclang-dev
+      - uses: fiam/arm-none-eabi-gcc@v1
+        with:
+          release: "9-2020-q2"
+
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: Check
+        run: |
+          cargo check
+          cargo check --all-features
+
+      - name: Build
+        run: cargo build --release
```

Comment:
Shouldn't this need `--target ${{ matrix.target }}`

Reply:
Good catch, deleted too much when copying the file.

Reply:
Should be fixed now.

Reply:
Is it? The logs for the latest CI "build (thumbv7em-none-eabi)" still use `/home/runner/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/bin/rustc`

Reply:
Oh nevermind I was confused because 90% of the lines are for build deps that therefore need to be compiled for the host target.

Reply:
Hmm, but `--target thumbv7em-none-eabi` is set as expected.